lib/chromedriver.js

```diff
@@ -4,42 +4,145 @@ import events from 'events';
 import { JWProxy } from 'appium-base-driver';
 import cp from 'child_process';
 import { system, fs, logger } from 'appium-support';
-import { retryInterval } from 'asyncbox';
-import { SubProcess } from 'teen_process';
+import { retryInterval, asyncmap } from 'asyncbox';
+import { SubProcess, exec } from 'teen_process';
 import B from 'bluebird';
-import { getChromedriverBinaryPath } from './install';
+import { getChromedriverBinaryPath, getChromedriverDir } from './install';
+import { getChromeVersion } from './utils';
+import semver from 'semver';
+import _ from 'lodash';
 
 
 const log = logger.getLogger('Chromedriver');
 
 const DEFAULT_HOST = '127.0.0.1';
 const DEFAULT_PORT = 9515;
+const CHROMEDRIVER_CHROME_MAPPING = {
```

Comment:
we are also looking for a solution which we can use to batch download the drivers and bundle it into a docker image

Reply:
That can be done if you download them somewhere and then pass in the executable bin, which will eventually be a capability for the android/uiautomator2 drivers.